.drone.yml

```diff
@@ -5632,3 +5632,154 @@ depends_on:
+        path: /identities
+      - name: snapshots
+        path: /snapshots
+    commands:
+      - tezos-node config init --network florencenet
+      - cp /identities/identity_1.json $${TEZOS_NODE_DIR}/identity.json
+      - tezos-node snapshot import /snapshots/octez-009-florence-1000.snapshot --block BKqNZHy5CFYLD6v6BjDLrEavrkK4qH39hhrsD7QtbWcdnnwoYub
+      - tezos-node run --no-bootstrap-peers
+
+  - name: tezedge-new-node
+    image: tezedge/tezedge-ci-builder
+    user: root
+    volumes:
+      - name: identities
+        path: /identities
+      - name: tools
+        path: /tools
+    commands:
+      - OCTEZ_IP=$(perl -e 'use Socket; my $a = gethostbyname("octez-node"); print inet_ntoa($a)')
+      - echo $${OCTEZ_IP}
+      # use alloc instead of jemalloc
+      - awk '/allocator/ { print "//" $0; next } /.*/ { print }' light_node/src/main.rs > /tmp/main.rs && mv /tmp/main.rs light_node/src/main.rs
+      - cargo build --package light-node --package protocol-runner
+      - export LD_LIBRARY_PATH=tezos/sys/lib_tezos/artifacts/
+      # install valgrind
+      - apt-get install -y valgrind
+      - >
+        valgrind --tool=massif --massif-out-file=new-node.massif.out
+        target/debug/light-node --protocol-runner target/debug/protocol-runner
+        --network florencenet --peers $${OCTEZ_IP}:9732
+        --config-file light_node/etc/tezedge/tezedge_drone.config
+        --identity-file /identities/identity_2.json
+        --peer-thresh-low 1 --peer-thresh-high 1
+        --init-sapling-spend-params-file ./tezos/sys/lib_tezos/artifacts/sapling-spend.params
+        --init-sapling-output-params-file ./tezos/sys/lib_tezos/artifacts/sapling-output.params &
+        echo $! > /var/run/tezedge.pid
+      - |
+        sh -c '
+        block=0
+        while [ $block -lt 1000 ]; do
+          sleep 5
+          b=$(curl -s localhost:18732/chains/main/blocks/head | jq .header.level)
+          block=$${b:-$block}
+          echo "===> Block level $block"
+        done
+        '
+      - echo "Bootstrapped. Killind the node and waiting for 10 seconds."
+      - kill -INT $(cat /var/run/tezedge.pid) && sleep 10
+      - ms_print new-node.massif.out > new-node.msp.out
+
+  - name: tezedge-old-node
+    image: tezedge/tezedge-ci-builder
+    user: root
+    volumes:
+      - name: identities
+        path: /identities
+      - name: tools
+        path: /tools
+    commands:
+      - OCTEZ_IP=$(perl -e 'use Socket; my $a = gethostbyname("octez-node"); print inet_ntoa($a)')
+      - git reset --hard origin/${DRONE_TARGET_BRANCH}
+      - rm -rf ./tezos-node-data ./light-node-data
+      # use alloc instead of jemalloc
+      - awk '/allocator/ { print "//" $0; next } /.*/ { print }' light_node/src/main.rs > /tmp/main.rs && mv /tmp/main.rs light_node/src/main.rs
+      - cargo clean && cargo build --package light-node --package protocol-runner
+      - export LD_LIBRARY_PATH=tezos/sys/lib_tezos/artifacts/
+      # install valgrind
```

Comment:
@akoptelov 
please remove valgrind installation,
and also from `run-dhat` step `apt-get update && apt-get install -y valgrind`
I pushed valgrind instalation to `tezedge/tezedge-ci-builder:latest`  so we dont need to install it everytime,
in 8 minutes, it will be tezedge-ci-builder:latest pushed to dockerhub